### ISSUE TITLE
Sentence-transformers with 'MPS' GPU

### ISSUE THREAD

**Issue by dbl001** [UKPLab/sentence-transformers#1564]
PyTorch 1.12 now supports the 'MPS' backend (Apple Metal).
Can I get sentence_transformers to use 'MPS' for embedding, etc.?

**Comment by SuperKenVery** [comment]
I believe it's solved by https://github.com/UKPLab/sentence-transformers/pull/2342 right?

**Comment by tomaarsen** [comment]
That is correct, this should be automatically supported now, so I'll be closing this. Thank you for bringing this to my attention @SuperKenVery and thank you for opening @dbl001.

- Tom Aarsen
